From f13d6f5241b504acee61b5874e2951d46cd1c60b Mon Sep 17 00:00:00 2001
From: SquallATF <squallatf@gmail.com>
Date: Fri, 31 Oct 2025 14:21:38 +0800
Subject: [PATCH] fix(windows): set manifest resource ID to 1 in nvim.rc for
 MinGW

On Windows, the main application manifest should use resource ID 1 (RT_MANIFEST).
This change updates `nvim.rc` to use `1 RT_MANIFEST nvim.manifest` instead of `2`,
ensuring the manifest is correctly embedded and recognized by the system.
---
 src/nvim/os/nvim.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/nvim/os/nvim.rc b/src/nvim/os/nvim.rc
index e838c93c16bbb0..74a34d382bc580 100644
--- a/src/nvim/os/nvim.rc
+++ b/src/nvim/os/nvim.rc
@@ -1,2 +1,2 @@
 #include "winuser.h"
-2 RT_MANIFEST nvim.manifest
+1 RT_MANIFEST nvim.manifest
